src/test/java/bio/terra/service/filedata/azure/tables/TableDirectoryDaoConnectedTest.java

```diff
@@ -84,7 +85,9 @@ public void cleanup() throws Exception {
   @Test
   public void testStorageTableMetadataDuringFileIngest() {
     // Test re-using same directory path, but for different files
-    String sharedTargetPath = "/test/path/";
+    String sharedParentDir = UUID.randomUUID().toString();
+    String sharedChildDir = UUID.randomUUID().toString();
+    String sharedTargetPath = String.format("/%s/%s/", sharedParentDir, sharedChildDir);
```

Comment:
Nice! If I'm reading these code changes correctly - this is the main change of this PR - to randomize the directory names so that we can be more certain that we're correctly testing these scenarios.

Reply:
That's right! It also checks that the directory entries for "empty" directories are deleted (not just the file's directory entry)